app/main.js

```diff
@@ -9,9 +11,20 @@ let win;
 
 
 const startRestAPI = (callback) => {
-    // find the python for this process                  This is literally the worst idea.
-    const python = which.sync('python', { all: true }).find(path => path.search('conda') !== -1);
-    const api = spawn(python, ['-m', 'qiime_studio']);
+    // find the conda python and use that to infer the conda bin directory
+    // which needs to be reapplied to the front of our path so that subprocess
+    // spawns work correctly.
+    const python = which.sync('python', { all: true })
+                        .find(binaryPath => binaryPath.search('conda') !== -1);
+    const condaBin = path.dirname(python);
+
+    const api = spawn('python', ['-m', 'qiime_studio'], {
```

Comment:
Should this be a hard coded 'python' or using the python path from Line 17 like before?